screwdriver.yaml

```diff
@@ -8,6 +8,11 @@ jobs:
         requires: [~pr, ~commit]
         steps:
             - gover: go version
+            - tidy: |
+                go mod tidy
+                echo 'If failure by this step, reason is including not used module in go.mod file.'
+                echo 'Retry to after the using "go mod tidy" command.'
```

Comment:
small suggestion
```suggestion
                echo 'This step failed because there is an unused module in the go.mod file.'
                echo 'Retry again after using the "go mod tidy" command.'
```

Reply:
Thank you.